pkg/tests/tests.go

```diff
@@ -108,3 +117,35 @@ func resourceReviewbody() []byte {
 	}
 	return buff.Bytes()
 }
+
+func clusterAuthorizationsBody() []byte {
+	buff := &bytes.Buffer{}
+	reservedResource := acctmgmtv1.NewReservedResource()
+
+	clusterAuthReq, err := acctmgmtv1.NewClusterAuthorizationRequest().
+		AccountUsername(helpers.AccountUsername).
+		AvailabilityZone("???").
+		BYOC(true).
+		CloudAccountID("???").
+		CloudProviderID("???").
+		ClusterID("???").
+		Disconnected(true).
+		DisplayName("???").
+		ExternalClusterID("???").
+		Managed(true).
+		ProductCategory("assistedInstall").
+		ProductID("osd").
+		Reserve(true).
+		Resources(reservedResource).
```

Comment:
OCP plans do not require resources to be sent.  OSD plans require resources.

See our integration test of OCP AssistedInstaller:  https://gitlab.cee.redhat.com/service/uhc-account-manager/-/blob/master/test/integration/cluster_authorization_test.go#L3206